src/test/java/org/jenkinsci/plugins/oic/OicSecurityRealmFipsTest.java

```diff
@@ -16,81 +17,75 @@
-        FormValidation response = descriptor.doCheckDisableSslVerification(true);
-        assertThat(
-                "States SSL verification can not be disabled",
-                response,
-                allOf(
-                        hasKind(FormValidation.Kind.ERROR),
-                        withMessageContaining("SSL verification can not be disabled")));
-        response = descriptor.doCheckDisableSslVerification(false);
-        assertThat("Validation is ok", response, hasKind(FormValidation.Kind.OK));
+        try (MockedStatic<FIPS140> fips140Mocked = mockStatic(FIPS140.class)) {
+            fips140Mocked.when(FIPS140::useCompliantAlgorithms).thenReturn(true);
+            OicSecurityRealm.DescriptorImpl descriptor = new OicSecurityRealm.DescriptorImpl();
+            FormValidation response = descriptor.doCheckDisableSslVerification(true);
+            assertThat(
+                    "States SSL verification can not be disabled",
+                    response,
+                    allOf(
+                            hasKind(FormValidation.Kind.ERROR),
+                            withMessageContaining("SSL verification can not be disabled")));
+            response = descriptor.doCheckDisableSslVerification(false);
+            assertThat("Validation is ok", response, hasKind(FormValidation.Kind.OK));
 
-        response = descriptor.doCheckDisableTokenVerification(true);
-        assertThat(
-                "States token verification can not be disabled",
-                response,
-                allOf(
-                        hasKind(FormValidation.Kind.ERROR),
-                        withMessageContaining("Token verification can not be disabled")));
-        response = descriptor.doCheckDisableTokenVerification(false);
-        assertThat("Validation is ok", response.kind, is(FormValidation.Kind.OK));
+            response = descriptor.doCheckDisableTokenVerification(true);
+            assertThat(
+                    "States token verification can not be disabled",
+                    response,
+                    allOf(
+                            hasKind(FormValidation.Kind.ERROR),
+                            withMessageContaining("Token verification can not be disabled")));
+            response = descriptor.doCheckDisableTokenVerification(false);
+            assertThat("Validation is ok", response.kind, is(FormValidation.Kind.OK));
+        }
     }
 
     @Test
     @LocalData
     void worksOnMigrationWithValidValuesTest(JenkinsRule j) {
-        assertThat("Instance is up and running with no errors", j.jenkins.getInitLevel(), is(InitMilestone.COMPLETED));
+        try (MockedStatic<FIPS140> fips140Mocked = mockStatic(FIPS140.class)) {
+            fips140Mocked.when(FIPS140::useCompliantAlgorithms).thenReturn(true);
+            assertThat(
+                    "Instance is up and running with no errors", j.jenkins.getInitLevel(), is(InitMilestone.COMPLETED));
```

Comment:
I don't think this is working like you expect it to.

the flag is used when Jenkins starts which is before we enter this method.  Additionally the mocking here is thread local, non local threads (like the numerous Jenkins startup threads) would not see this.